### PR TITLE
password reset error bugfix

### PR DESCRIPTION
**Pull request by arfacamble**
avoid error if email is nil

sentry error alert from production
https://dluhc-core.sentry.io/issues/4211300771/?referrer=slack

### CODE REVIEW

**Review by RachaelBooth — COMMENTED** [review]
Looks sensible to me. 

@arfacamble Is there a ticket reference or anything that should be in the PR name before merge? (to help anyone looking back at commits). I'm happy to approve as is if there isn't.

File: app/controllers/auth/passwords_controller.rb
Comment (jamesrose) [inline app/controllers/auth/passwords_controller.rb:7]: Is there a spec for this already? If not, could you please add one?

**Comment by arfacamble** [comment]
> 

apologies slow reply @RachaelBooth  this slipped off my radar.
there is no ticket for this. There was a sentry alert but it was a while ago now so I have lost it.
Yes @jamesrose  I will write a test for this. It was a production bug so I chucked somethign up quick in case it recurred and then slightyl forgot about it

**Review by jamesrose — APPROVED** [review]

**Pull request merged**
